Managing OSTree content in Content Management Guide (#1024)
theforeman/foreman-documentation: guides/common/modules/con_managing-ostree-content.adoc

[id="con_managing-ostree-content_{context}"]
= Managing OSTree Content

*OSTree* is a tool to manage bootable, immutable, versioned file system trees.

> TODO why

Reviewer: I feel that you don't necessarily need to answer this question. 
We don't need to convince anyone to manage OSTree content in Katello. 
We just need to tell them how.

[id="con_managing-ostree-content_{context}"]
= Managing OSTree Content

*OSTree* is a tool to manage bootable, immutable, versioned file system trees.

OSTree makes it easy to install and update Linux-based operating systems on hosts and to switch between versions of operating systems on hosts.
If a host update fails, you can easily revert the operating system on the host to the previous working version.

You can create an OSTree image using an image builder and expose it in an OSTree repository on an HTTP server.
Then you can use your {Project} to synchronize and manage OSTree branches from the exposed OSTree repository.

.Prerequisites
* In {ProjectServer}, OSTree management tools are enabled by default.

> If you ever have a reason to enable the tool, enter the following command:

Reviewer 2: You might mention that it is el8 only.  (doesn't work on el7)

Author: Thanks for catching that! I intended, but forgot :)

> 

Reviewer 3: ```suggestion
+
```

Note that the source code block is not part of the list.

[id="con_managing-ostree-content_{context}"]
= Managing OSTree Content

You can use OSTree to manage bootable, immutable, versioned file system trees.

OSTree makes it easy to install and update Linux-based operating systems on hosts and to switch between versions of operating systems on hosts.
If a host update fails, you can easily revert the operating system on the host to the previous working version.

You can create an OSTree image using an image builder and expose it in an OSTree repository on an HTTP server.
Then you can use your {Project} to synchronize and manage OSTree branches from the exposed OSTree repository.

.Prerequisites

> * OSTree content is supported only when running {Project} on RHEL 8.

Reviewer 3: ```suggestion
* OSTree content is supported only when running {Project} on {RHEL} 8.
```

Does is also work with CentOS 8? See https://docs.theforeman.org/nightly/Installing_Server_on_Red_Hat/index-foreman-el.html#supported-operating-systems_foreman

Author: It actually does (confirmed with devs)! Good catch. Please check if you're okay with the condition.

> * OSTree content is supported in {Project} only when running on RHEL 8.

Author: @jlsherrill @melcorr @maximiliankolb Are we okay with the phrasing here?

Author: Maybe now it's better.

Reviewer: Fine by me!

* In {ProjectServer}, OSTree management tools are enabled by default.
If you ever have a reason to enable the tool, enter the following command:
+
[options="nowrap" subs="+quotes,attributes"]
----
# {foreman-installer} --foreman-proxy-content-enable-ostree
----

> * If you want to use Hammer CLI, the `hammer-cli-katello` plugin must be enabled on {ProjectServer}.

Reviewer 3: ```suggestion
* If you want to use Hammer CLI, you must enable the `hammer-cli-katello` plugin on {ProjectServer}.
```
